smurfs/src/components/App.css

```diff
@@ -1,28 +1,320 @@
 .App {
+  width: 100%;
+  margin: 0 auto;
+  display: flex;
+  flex-direction: column;
   text-align: center;
 }
 
-.App-logo {
-  animation: App-logo-spin infinite 20s linear;
-  height: 80px;
+.smurf-family {
+  position: absolute;
+  z-index: 400;
+  width: 800px;
+  margin: 0 auto;
+  left: 0;
+  right: 0;
+  top: 76%;
+  position: fixed;
+  user-select: none;
+  pointer-events: none;
+}
+
+.background2 {
+  width: 100%;
+  height: 50%;
+  background: rgb(223, 191, 52);
+  position: absolute;
+  left: 0;
+  top: 49%;
+  z-index: 200;
+  position: fixed;
+}
+
+.background {
+  width: 100%;
+  height: 50%;
+  background: rgb(233, 222, 67);
+  background: -moz-radial-gradient(
+    circle,
+    rgba(233, 222, 67, 1) 45%,
+    rgba(255, 209, 0, 1) 100%
+  );
+  background: -webkit-radial-gradient(
+    circle,
+    rgba(233, 222, 67, 1) 45%,
+    rgba(255, 209, 0, 1) 100%
+  );
+  background: radial-gradient(
+    circle,
+    rgba(233, 222, 67, 1) 45%,
+    rgba(255, 209, 0, 1) 100%
```

Comment:
excellent use of radial gradients and its properties, such as 'circle'
great use of rgba 's as well for opacity effects - made it really sharp